### PR TITLE
Feature: Store sett rewards to dynamo table

### LINKED ISSUE #387: Post JSON of Sett Distribution to github
On each approve, we should post the breakdown of distribution by vault to a publicly accessible github in the below format allowing for public audit of our rewards distributions:

```
{
   sett: {
      boosted: {
         BADGER: amount
      }
      unboosted: {
         cvxCRV: amount
      }
   }
   ...
}
```

### PR DESCRIPTION
**Pull request by SHAKOTN**
1. Add `put_rewards_data` dynamo handler
2. Use handler in `approve_root` in case of `success=True`
3. Many tests for all tastes

Also, minor fixes and cleanups

Fixes #387

### CODE REVIEW

File: rewards/classes/RewardsManager.py [hunk 1]
```diff
@@ -157,11 +163,11 @@ def calculate_all_sett_rewards(
         if len(invalid_totals):
             self.report_invalid_totals(invalid_totals)
             
-        return combine_rewards(all_rewards, self.cycle)
+        return combine_rewards(all_rewards, self.cycle), rewards_analytics
     
     def report_invalid_totals(self, invalid_totals: List[List[str]]) -> None:
         send_plain_text_to_discord(
-            msg=f"INCORRECT REWARDS DISTRIBTION {DiscordRoles.RewardsPod}",
+            message=f"INCORRECT REWARDS DISTRIBTION {DiscordRoles.RewardsPod}",
```
Comment (btcookies) [inline rewards/classes/RewardsManager.py:170]: does this need to be updated in function?

File: rewards/aws/helpers.py [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 logger = logging.getLogger("aws-helpers")
 
 DYNAMO_ASSUME_ROLE = "arn:aws:iam::784874126256:role/k8s-bots"
 
+
 def get_metadata_table(production: bool):
     return "metadata-prod" if production else "metadata-staging"
 
 
+def get_rewards_table(production):
```
[the remark this answers is not in the document]
Comment (SHAKOTN) [reply rewards/aws/helpers.py]: Fixed

**Review by btcookies — APPROVED** [review]
LGTM

**Review by btcookies — APPROVED** [review]
LGTM

**Pull request merged**